app/controllers/authenticated_controller.rb

```diff
@@ -19,6 +19,13 @@ def require_authentication
     redirect_to TradeTariffDevHub.identity_consumer_url, allow_other_host: true
   end
 
+  def set_paper_trail_whodunnit
```

Comment:
How come you're overriding papertrail?

Is this leftover from before the rebase?